sw/device/lib/dif/dif_clkmgr.c

```diff
@@ -227,6 +227,43 @@ dif_result_t dif_clkmgr_measure_ctrl_get_enable(const dif_clkmgr_t *clkmgr,
   return kDifOk;
 }
 
+// The earlgrey and englishbreakfast CSR differences mean they have a
+// different number of clock measuremen units. The ideal way to handle
+// this difference is just generating the dif code, and we intend to do
+// that with multi-top. However, for the time being we need to rely on
+// macro trickery, just generating code for a specific measurement if
+// there is a #define for the corresponding enable CSR offset.
+//
+// These macro are a tricky way to implement a macro like the following:
+// #define CONDITIONALLY_DO	\
+//   #ifdef SOMETHING do_something \
+//   #else do_something_else       \
+//   #endif
+// However, such macros are illegal in C. So the macros below implement
+// the conditional using a suggestion presented in
+// https://stackoverflow.com/questions/72266480/can-ifdef-be-used-inside-a-macro
+//
+// The reason we need these macros is to handle differences between earlgrey
+// and englishbreakfast without generating the dif code.
```

Comment:
Is there maybe an issue we can associate with this as part of a `TODO(...)` comment here?
It would be nice to remove this workaround sometime in the future once `englishbreakfast` is a native top-level...